templates/thank-you.html

```diff
@@ -32,4 +32,24 @@ <h1 data-qa="title">{{ _("Thank you for completing the {survey_title}").format(
     <a href="https://www.ons.gov.uk/surveys">https://www.ons.gov.uk/surveys</a></p>
   {% endif %}
 
+  {% if content.view_submitted_response.enabled %}
+    <h1 data-qa="view-answers-title">{{ _("Get a copy of your answers") }}</h1>
+    {% if content.view_submitted_response.expired == false %}
+      <p class="u-mb-s">{{ _("You can <a href='{url}' id='view-answers-link'>save or print your answers</a> for your records").format(url = content.view_submitted_response.url) }}</p>
```

Comment:
The screenshots on the card have a full stop
```suggestion
      <p class="u-mb-s">{{ _("You can <a href='{url}' id='view-answers-link'>save or print your answers</a> for your records.").format(url = content.view_submitted_response.url) }}</p>
```